crates/nix_health/src/lib.rs

```diff
@@ -134,17 +135,29 @@ pub async fn run_checks_with(flake_url: Option<FlakeUrl>) -> anyhow::Result<Vec<
         .await
         .as_ref()
         .with_context(|| "Unable to gather nix info")?;
-    let action_msg = format!(
-        "🩺️ Checking the health of your Nix setup ({} on {})",
-        &nix_info.nix_config.system.value, &nix_info.nix_env.os
+
+    let sys_info_msg = format!(
+        "\n  - System: {}\n  - OS: {}{}",
+        &nix_info.nix_config.system.value,
+        &nix_info.nix_env.os,
+        (if let NixInstaller::DetSys { .. } = &nix_info.nix_env.installer {
+            format!("\n  - Installer: {}\n", &nix_info.nix_env.installer)
+        } else {
+            "".to_string()
+        })
```

Comment:
Let's _always_ printer the installer (even if it is `Other`).

Reply:
And this should be skipped on NixOS right?

Reply:
https://github.com/juspay/omnix/pull/213/commits/b101353a7b96d4efcd24b9c407e79813327d6473